logdevice/server/Listener.h

```diff
@@ -134,6 +134,9 @@ class Listener : public folly::AsyncServerSocket::AcceptCallback {
   // EventLoop on which listener is running
   KeepAlive loop_;
 
+  // Whether to use DSCP value provided by client or server default
```

Comment:
Apparently servers use zero (not their default) if DSCP is disabled.

Reply:
I thnk this is because the default value is 0.
`init("server-default-dscp",
       &server_dscp_default,
       "0",
       parse_validate_range<uint8_t>(0, 63),
       "Use default DSCP to setup to server sockets at Sender."
       "Range was defined by https://tools.ietf.org/html/rfc4594#section-1.4.4",
       SERVER | REQUIRES_RESTART,
       SettingsCategory::Configuration);`

Reply:
I remember when testing it we noticed that even if the default wasn't zero, the server was responding with zero if reflection is disabled. Right?